superdesk/io/registry.py

```diff
@@ -115,3 +118,29 @@ def service(service_id):
         return ListCursor(
             [service(_id) for _id in registered_feeding_services]
         )
+
+
+def register_webhook(provider_name, webhook_name, webhook_service=None):
```

Comment:
actually why would we create multiple resources for this? couldn't be provider a param as well?
`/webhook?provider=woofoo`